nixos/modules/services/backup/znapzend.nix

```diff
@@ -24,8 +44,9 @@ in
         path = with pkgs; [ zfs mbuffer openssh ];
 
         serviceConfig = {
-          ExecStart = "${pkgs.znapzend}/bin/znapzend";
+          ExecStart = "${pkgs.znapzend}/bin/znapzend --logto=${cfg.logTo} --loglevel=${cfg.logLevel} ${optionalString cfg.noDestroy "--nodestroy"}";
           ExecReload = "${pkgs.coreutils}/bin/kill -HUP $MAINPID";
+          #Restart = "on-failure";
```

Comment:
Accident, fixed in the [next commit](https://github.com/NixOS/nixpkgs/pull/25960/commits/c0ee81fcd25c0391cd129e4b33c3ca47104b9cf7)